docs/AnaplanExport-batchsource.md

```diff
@@ -0,0 +1,139 @@
+# Anaplan source
+
+Description
```

Comment:
Just by reading this file, it's not clear what the all the different properties are referring to?  Should we give more context?

Reply:
The context is added. Thanks!